js/apps/admin-ui/src/realm-settings/realm-settings-section.css

```diff
@@ -192,15 +192,15 @@ article.pf-c-card.pf-m-flat.kc-email-connection
 }
 
 .kc-emptyExecutors {
-  color: #8d9195;
+  color: #717477;
```

Comment:
Perhaps we can use some of the pre-defined PatternFly variables for colors here?

Reply:
yeah, sure, great idea @jonkoops